tests/maintel/test_setup_whitelight_flats.py

```diff
@@ -0,0 +1,148 @@
+        self.script.electrometer.evt_summaryState.aget = unittest.mock.AsyncMock(
+            return_value=salobj.State.ENABLED
+        )
+        self.script.electrometer.evt_summaryState.summaryState = (
+            unittest.mock.AsyncMock(return_value=salobj.State.ENABLED)
+        )
+        self.script.fiberspec_red = unittest.mock.AsyncMock()
+        self.script.fiberspec_red.evt_summaryState.aget = unittest.mock.AsyncMock(
+            return_value=salobj.State.ENABLED
+        )
+        self.script.fiberspec_red.evt_summaryState.summaryState = (
+            unittest.mock.AsyncMock(return_value=salobj.State.ENABLED)
+        )
+        self.script.fiberspec_blue = unittest.mock.AsyncMock()
+        self.script.fiberspec_blue.evt_summaryState.aget = unittest.mock.AsyncMock(
+            return_value=salobj.State.ENABLED
+        )
+        self.script.fiberspec_blue.evt_summaryState.summaryState = (
+            unittest.mock.AsyncMock(return_value=salobj.State.ENABLED)
+        )
+        self.script.linearstage_led_focus = unittest.mock.AsyncMock()
+        self.script.linearstage_led_focus.evt_summaryState.aget = (
+            unittest.mock.AsyncMock(return_value=salobj.State.ENABLED)
+        )
+        self.script.linearstage_led_focus.evt_summaryState.summaryState = (
+            unittest.mock.AsyncMock(return_value=salobj.State.ENABLED)
+        )
+        self.script.linearstage_led_select = unittest.mock.AsyncMock()
+        self.script.linearstage_led_select.evt_summaryState.aget = (
+            unittest.mock.AsyncMock(return_value=salobj.State.ENABLED)
+        )
+        self.script.linearstage_led_select.evt_summaryState.summaryState = (
+            unittest.mock.AsyncMock(return_value=salobj.State.ENABLED)
+        )
+        self.script.linearstage_projector_select = unittest.mock.AsyncMock()
+        self.script.linearstage_projector_select.evt_summaryState.aget = (
+            unittest.mock.AsyncMock(return_value=salobj.State.ENABLED)
+        )
+        self.script.linearstage_projector_select.evt_summaryState.summaryState = (
+            unittest.mock.AsyncMock(return_value=salobj.State.ENABLED)
+        )
+        self.script.led_projector = unittest.mock.AsyncMock()
+        self.script.led_projector.evt_summaryState.aget = unittest.mock.AsyncMock(
+            return_value=salobj.State.ENABLED
+        )
+        self.script.led_projector.evt_summaryState.summaryState = (
+            unittest.mock.AsyncMock(return_value=salobj.State.ENABLED)
+        )
+        self.script.mtcalsys.tunablelaser = unittest.mock.AsyncMock()
+        self.script.mtcalsys.tunablelaser.evt_summaryState.aget = (
+            unittest.mock.AsyncMock(return_value=salobj.State.ENABLED)
+        )
+        self.script.mtcalsys.tunablelaser.evt_summaryState.summaryState = (
+            unittest.mock.AsyncMock(return_value=salobj.State.ENABLED)
+        )
+
+    async def test_configure(self):
+        async with self.make_script():
```

Comment:
See my suggestion of calling the mock setup during `make_script`. The way you are running this right now it is not going to do the mocking, and thus will create MTCalsys during these tests... This put an additional strain on the test kafka cluster.